pkg/gofr/migration/clickhouse_test.go

```diff
@@ -108,12 +108,3 @@ func Test_ClickHouseBeginTransaction(t *testing.T) {
 
 	assert.Contains(t, logs, "Clickhouse Migrator begin successfully")
 }
-
-func Test_ClickHouseRollback(t *testing.T) {
```

Comment:
Needed to remove the test since now we are using `fatalf` if migration is failed and test will exit immidiately. Also this code is covered in `TestMigrationRunClickhouseMigrationFailure` test method.